homeassistant/components/group/media_player.py

```diff
@@ -405,7 +405,12 @@ def async_update_state(self) -> None:
         else:
             off_values = {MediaPlayerState.OFF, STATE_UNAVAILABLE, STATE_UNKNOWN}
             if states.count(states[0]) == len(states):
-                self._state = states[0]
+                try:
+                    self._state = MediaPlayerState(states[0])
```

Comment:
This can potentially be a breaking change. Can we keep track of a set of unknown states and for each unknown state print a warning once?

Reply:
Done